26-map-filter-reduce/solution.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.map = function(arr, func) {
```

Comment:
Generally these methods should use callbacks that accept (value, index, array). Check the mdn docs for full specs